subsys/bluetooth/controller/Kconfig

```diff
@@ -887,7 +887,7 @@ endif # BT_CTLR_SYNC_PERIODIC
 config BT_CTLR_SYNC_TRANSFER_SENDER
 	bool "Periodic Advertising Sync Transfer sender"
 	depends on BT_CTLR_SYNC_TRANSFER_SENDER_SUPPORT
-	select BT_TICKER_LAZY_GET
+	select BT_TICKER_LAZY_GET if BT_LL_SW_SPLIT
```

Comment:
Already covered here: https://github.com/zephyrproject-rtos/zephyr/blob/593716d501d3087a30d136ef598264657e79a3ac/subsys/bluetooth/controller/Kconfig.ll_sw_split#L69

```suggestion
```

Reply:
Actually, that is insufficient since it *only* applies for Nordic controllers. The original change is correct

Reply:
But @Tronil , dont you have a custom choice on the same line as `config BT_LLL_VENDOR_NORDIC/OPENISA`  where this `select` be present?

Reply:
We could have (we don't at the moment), but why not just have it in the one place instead of once per vendor?

Reply:
I will revert to the original solution then

Reply:
> We could have (we don't at the moment), but why not just have it in the one place instead of once per vendor?

Just hopping to keep the specifics inside ll_sw_split Kconfig file. Not able to think of an alternative for now, approving the current change in this PR.